frontend/device/kindle/device.lua

```diff
@@ -918,12 +921,16 @@ function KindleOasis:init()
         }
     }
 
+    Kindle.init(self)
+
+    --- @note See comments in KindleOasis2:init() for details.
     local haslipc, lipc = pcall(require, "liblipclua")
     if haslipc and lipc then
         local lipc_handle = lipc.init("com.github.koreader.screen")
         if lipc_handle then
             local orientation_code = lipc_handle:get_string_property(
                 "com.lab126.winmgr", "accelerometer")
+            logger.dbg("orientation_code = ", orientation_code)
```

Comment:
That's not a bad idea at all ;).

But please add it to *all* the devices (it's missing from the Scribe).

You can also drop the space *after* the equal sign, a comma already ensures whitespace ;).

Reply:
> That's not a bad idea at all ;).
I am still hoping for people with Oases to come back with testing xD

> it's missing from the Scribe
Lol, overlooked that one, I have it on my Scribe though.

> a comma already ensures whitespace
Oh, that's why! Thanks :D

Reply:
> (it's missing from the Scribe)

Hmm, I double-check and that `logger.dbg` is there for Scribe too (as well as for all Oases)! Did you mean something else?

Reply:
Nope, must be getting cross-eyed, my bad ;p.